app/keepers/keepers.go

```diff
@@ -563,6 +570,13 @@ func NewAppKeepers(
 		govModAddress,
 	)
 
+	appKeepers.ClockKeeper = clockkeeper.NewKeeper(
+		appKeepers.keys[clocktypes.StoreKey],
+		appCodec,
+		*appKeepers.ContractKeeper,
+		govModAddress,
```

Comment:
Do we want a SubDAO to control contracts instead of Gov? (Quicker, techy people vs just token holders)

Reply:
Yes! This would be a great improvement.

Reply:
You can implement an allowlist similar to x/drip

Reply:
I agree

I think for now we should just use standard Gov account then figure out the proper SubDAO (mainnet, and testnet?) in a v2 of the module